packages/widgets/src/lib/components/InferenceWidget/shared/WidgetFooter/WidgetFooter.svelte

```diff
@@ -28,18 +29,26 @@
 			JSON Output
 		</button>
 	{/if}
-	<button
-		class="ml-auto flex items-center"
-		type="button"
-		on:click|preventDefault={() => updateWidgetState(model.id, "isMaximized", !isMaximized)}
-	>
-		<IconMaximize classNames="mr-1" />
-		{#if !isMaximized}
-			Maximize
-		{:else}
-			Minimize
-		{/if}
-	</button>
+	<LogInPopover bind:open={popOverOpen} classNames="ml-auto">
+		<button
+			class="flex items-center"
+			type="button"
+			on:click|preventDefault={() => {
+				if (!$isLoggedIn) {
+					popOverOpen = true;
+					return;
+				}
+				updateWidgetState(model.id, "isMaximized", !isMaximized);
+			}}
+		>
+			<IconMaximize classNames="mr-1" />
+			{#if !isMaximized}
+				Maximize
+			{:else}
+				Minimize
+			{/if}
+		</button>
+	</LogInPopover>
```

Comment:
What's the reason behind this? (preventing logged out users from maximizing the widget)

Reply:
yes

Reply:
I'm not convinced we should do that, but OK (I am probably missing context here)